ui/apps/everest/src/components/cluster-form/resources/messages.tsx

```diff
@@ -4,6 +4,10 @@ export const Messages = {
     min: (val: number) => `The value cannot be less than ${val}`,
     max: (val: number) => `The value cannot be more than ${val}`,
     odd: 'The value cannot be even',
+    numberOfShards: '№ of shards',
```

Comment:
what character is this? Seems different from standard ones

Reply:
good catch, Fabio! I used one from my keyboard, but i see we use another one Nº, so will change it

Reply:
fixed in https://github.com/percona/everest/pull/746/commits/e10cef70b608f1c8b6169813e10e2b54dac6b435